client/components/Card/Card.jsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react'
 import pt from 'prop-types'
 
+import { useSelector, useDispatch } from 'react-redux'
+
+import { updateCartProducts, createCart } from '../../redux/cart/actions'
+// import { createCart } from '../../api/cart'
 import './Card.scss'
 
 const Card = ({ item }) => {
-  const { name, image, price, description } = item
+  const { name, image, price, description, _id } = item
+
+  const dispatch = useDispatch()
+  const loading = useSelector((state) => state.cart.loading)
+  const cartId = useSelector((state) => state.cart.id)
+
+  const handleClick = (e, productId) => {
+    e.preventDefault()
+
+    if (loading) return
+
+    if (cartId) {
+      dispatch(updateCartProducts(productId))
+    } else {
+      dispatch(createCart(productId))
+    }
+  }
 
   return (
     <div className="card">
       <div className="card__image">
         <img src={image} alt={name} />
-        <button>+</button>
+        <button onClick={(e) => handleClick(e, _id)}>+</button>
```

Comment:
@AStashevskaya Try to avoid creating functions in `jsx`, you could use `<button onClick={handleClick}>+</button>` and read `_id` from `props` inside `handleClick`